### ISSUE TITLE
Make the Notification component accessible

### ISSUE THREAD

**Issue by dmytro-danylov** [wix/wix-style-react#1236]
### Summary:
The screen readers should inform the user when the notification component appears.

### Priority:
- [  ] Blocker
- [x] Urgent
- [  ] Normal
- [  ] Minor

### Expected behavior:
When a notification appears, screen readers should be able to detect it and read it to the user.

### Relevant Components:
Notification
